ddl/backfilling.go

```diff
@@ -56,8 +59,46 @@ const (
 	typeUpdateColumnWorker     backfillWorkerType = 1
 	typeCleanUpIndexWorker     backfillWorkerType = 2
 	typeAddIndexMergeTmpWorker backfillWorkerType = 3
+
+	// InstanceLease is the instance lease.
+	InstanceLease = 1 * time.Minute
 )
 
+// BackfillJob is for a tidb_ddl_backfill table's record.
+type BackfillJob struct {
+	ID            int64
+	JobID         int64
+	EleID         int64
+	EleKey        []byte
+	Tp            model.BackfillType
+	State         model.JobState
+	StoreID       int64
+	InstanceID    string
+	InstanceLease types.Time
+	Mate          *model.BackfillMeta
+}
+
+// AbbrStr returns the BackfillJob's info without the Mate info.
+func (bj *BackfillJob) AbbrStr() string {
+	return fmt.Sprintf("ID:%d, JobID:%d, EleID:%d, Type:%s, State:%s, InstanceID:%s, InstanceLease:%s",
+		bj.ID, bj.JobID, bj.EleID, bj.Tp, bj.State, bj.InstanceID, bj.InstanceLease)
+}
+
+// GetOracleTime returns the current time from TS.
+func GetOracleTime(store kv.Storage) (time.Time, error) {
+	currentVer, err := store.CurrentVersion(kv.GlobalTxnScope)
```

Comment:
Can we use txn.startTS to get one oracle time, instead of get time from currentVersion?

Reply:
This could reduce get ts from pd request.